### PR TITLE
Addition a form 'Contact Us', validation and display form data

### PR DESCRIPTION
**Pull request by LisKorzun**

### CODE REVIEW

File: app/Resources/views/base.html.twig [hunk 1]
```diff
@@ -13,7 +13,9 @@
     <div class="container">
         <div class="navbar-header">
             <a  href="http://www.it-academy.by/" target="_blank">
-                <img class="logo" alt="Brand" src="{{ asset('bundles/testlabwebsite/images/logo.png') }}">
+                {% image '@logo' %}
```
Comment (EduardTrayan) [inline app/Resources/views/base.html.twig]: You can just specify `output` value for images like for `css/js`

File: src/TestLab/WebsiteBundle/Controller/IndexController.php [hunk 2]
```diff
@@ -11,4 +14,34 @@
  */
 class IndexController extends AbstractController
 {
+    /**
+     * @Route("/", name="test_lab_website_createpage")
+     * @Template
+     */
+    public function createAction(Request $request)
+    {
+        $form = $this->getFormFactory()->create('website_contact');
```
Comment (EduardTrayan) [inline src/TestLab/WebsiteBundle/Controller/IndexController.php]: Please create helper method `createForm` in `AbstractController` and use it like in http://api.symfony.com/2.6/Symfony/Bundle/FrameworkBundle/Controller/Controller.html#createForm()

File: src/TestLab/WebsiteBundle/Controller/IndexController.php [hunk 3]
```diff
@@ -11,4 +14,34 @@
...
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
```
Comment (EduardTrayan) [inline src/TestLab/WebsiteBundle/Controller/IndexController.php]: You can just use `isValid`, because if form is not submitted, `isValid` will return false

File: src/TestLab/WebsiteBundle/Resources/config/routing.yml [hunk 4]
```diff
@@ -1,11 +1,3 @@
-test_lab_website_createpage:
```
Comment (EduardTrayan) [inline src/TestLab/WebsiteBundle/Resources/config/routing.yml]: Why did you remove static pages

File: src/TestLab/WebsiteBundle/Controller/IndexController.php [hunk 5]
```diff
@@ -11,4 +14,34 @@
...
+            $data = $form->getData();
+            $session = $request->getSession();
+            foreach($data as $key => $value){
+                $session->set($key, $value);
+            }
+            return new RedirectResponse('view');
```
Comment (EduardTrayan) [inline src/TestLab/WebsiteBundle/Controller/IndexController.php]: Please create helper method `redirect` in AbstractController and use it like in http://api.symfony.com/2.6/Symfony/Bundle/FrameworkBundle/Controller/Controller.html#redirect()

File: src/TestLab/WebsiteBundle/Form/ContactType.php [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+
+class ContactType extends AbstractType
+{
+    /**
+     * @param FormBuilderInterface $builder
+     * @param array $options
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
```
Comment (EduardTrayan) [inline src/TestLab/WebsiteBundle/Form/ContactType.php]: Please use `[]` instead of `array` at all places

File: src/TestLab/WebsiteBundle/Resources/views/Index/create.html.twig [hunk 7]
```diff
@@ -8,5 +8,36 @@
         <h1>Welcome</h1>
         <p>You can send us a message, as well as view all posts.</p>
     </div>
+    <div class="col-md-3"></div>
```
Comment (EduardTrayan) [inline src/TestLab/WebsiteBundle/Resources/views/Index/create.html.twig]: Please try to use just `{{ form(form) }}` instead of custom rendering of each field.

You can manipulate for view using http://symfony.com/doc/current/cookbook/form/form_customization.html

File: app/Resources/views/base.html.twig (hunk 1 above)
Comment (LisKorzun) [inline app/Resources/views/base.html.twig]: ок

File: src/TestLab/WebsiteBundle/Controller/IndexController.php (hunk 2 above)
Comment (LisKorzun) [inline src/TestLab/WebsiteBundle/Controller/IndexController.php]: ок

File: src/TestLab/WebsiteBundle/Controller/IndexController.php (hunk 3 above)
Comment (LisKorzun) [inline src/TestLab/WebsiteBundle/Controller/IndexController.php]: ок

File: src/TestLab/WebsiteBundle/Controller/IndexController.php (hunk 5 above)
Comment (LisKorzun) [inline src/TestLab/WebsiteBundle/Controller/IndexController.php]: ок

File: src/TestLab/WebsiteBundle/Form/ContactType.php (hunk 6 above)
Comment (LisKorzun) [inline src/TestLab/WebsiteBundle/Form/ContactType.php]: sorry, I forgot
fixed

File: src/TestLab/WebsiteBundle/Resources/views/Index/create.html.twig (hunk 7 above)
Comment (LisKorzun) [inline src/TestLab/WebsiteBundle/Resources/views/Index/create.html.twig]: ок

File: src/TestLab/CommonBundle/Controller/AbstractController.php [hunk 8]
```diff
@@ -32,4 +39,49 @@ public function getTemplate()
...
+    public function setFormFactory(FormFactory $formFactory)
+    {
+        $this->formFactory = $formFactory;
+    }
+
+    /**
+     * Get formFactory
+     * @return FormFactory
+     */
+    public function getFormFactory()
```
Comment (EduardTrayan) [inline src/TestLab/CommonBundle/Controller/AbstractController.php]: I think you should change visibility of this method to `protected`, because we'll not use this method outside our controllers, which will be children of this controller

File: src/TestLab/CommonBundle/Controller/AbstractController.php [hunk 9]
```diff
@@ -32,4 +39,49 @@ public function getTemplate()
...
+
+    /**
+     * Returns a RedirectResponse to the given URL.
+     *
+     * @param string $url    The URL to redirect to
+     * @param int    $status The status code to use for the Response
+     *
+     * @return RedirectResponse
+     */
+    public function redirect($url, $status = 302)
```
Comment (EduardTrayan) [inline src/TestLab/CommonBundle/Controller/AbstractController.php]: Same as `getFormFactory`

File: src/TestLab/CommonBundle/Controller/AbstractController.php (hunk 8 above)
Comment (LisKorzun) [inline src/TestLab/CommonBundle/Controller/AbstractController.php]: ок

File: src/TestLab/CommonBundle/Controller/AbstractController.php (hunk 9 above)
Comment (LisKorzun) [inline src/TestLab/CommonBundle/Controller/AbstractController.php]: ок
